.github/workflows/test.yml

```diff
@@ -203,14 +215,25 @@ jobs:
            COVERALLS=1 CONTAINER_RUNNABLE=1 make check
         popd
 
+    - name: Check custom OVN image build works
+      run: |
+        pushd dist/images
+          make fedora-dev
+        popd
```

Comment:
What is this for? Wouldn't if be more useful to actually be able to use this target elsewhere instead of hardcoding the docker build commands around?

Reply:
> What is this for?

This add some testing to github actions to check that we can build custom OVN images.

> Wouldn't if be more useful to actually be able to use this target elsewhere instead of hardcoding the docker build commands around?

You mean like replacing the "docker build" commands at test.yaml with `make fedora` ?

Reply:
yeah, or equivalent

Reply:
> yeah, or equivalent

That's out of the scope of this PR, I think, I can try at a follow up.

Reply:
it's not because if you did you would not need this extra step and the time it takes

Reply:
From previus comment, I can try to replace at test.yaml and kind.sh the docker build with `make fedora` using proper variables, make sense ?